app/views/actuals/uploads/new.html.haml

```diff
@@ -8,7 +8,13 @@
 
       = t("page_content.actuals.upload.copy_html", report_actuals_template_path: report_actuals_upload_path(@report_presenter, format: :csv))
 
-      = t("page_content.actuals.upload.warning_html")
+      .govuk-warning-text
```

Comment:
Should this use the partial?

Reply:
And should it be conditional?

Reply:
This is a different warning and it's on the `new` view rather than the `update` (error) one

Reply:
But I guess we could abstract the partial that gets added later into a generic warning component with the translation passed in, since that's the only thing that differs?

Reply:
Done the above!